### PR TITLE
Add Hangouts Chat version 19.1.281

### PR DESCRIPTION
**Pull request by compuguy**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/hangouts-chat.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'hangouts-chat' do
```
Comment (vitorgalvao) [inline Casks/hangouts-chat.rb]: I’ve [deleted the other `chat`](https://gsuite.google.com/products/chat/). Rename this one to `chat`.

File: Casks/hangouts-chat.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  version '19.1.281'
+  sha256 '30ac3007e565c8ac1f57c92291c9c58dec6a1d303eb19dd7c8c8cc080bb7949b'
+
+  url "https://dl.google.com/chat/#{version}/InstallHangoutsChat.dmg"
+  name 'Hangouts Chat'
```
Comment (vitorgalvao) [inline Casks/hangouts-chat.rb]:
```suggestion
  name 'Google Hangouts Chat'
```

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/hangouts-chat.rb (hunk 1 above)
Comment (compuguy) [reply Casks/hangouts-chat.rb]: Ok. Changing formula and file to chat.

**Comment by compuguy** [comment]
@vitorgalvao fixes are done. Should be good to go after travis-ci finishes.

**Comment by vitorgalvao** [comment]
Thank you for your contribution.

One small note. For future PRs, please leave the template with the default layout, as that makes it easier for us to parse.

Welcome!

**Pull request merged**